README.md

```diff
@@ -165,6 +165,7 @@ by default):
 | user           | enabled    | username and host (conditional on ssh status) |
 | venv           | enabled    | Python virtual environment                    |
 | working_dir    | enabled    | current working directory                     |
+| conda env	     | disabled   | Conda environment			                  |
```

Comment:
Please align markers.

Reply:
Allso move this un level up (i.e example below venv)

Reply:
the unaligned marker is shown only in git diff, it renders aligned both in my local machine as well as raw code in my forked repo,
also I have moved the conda env below venv as specified in previous review.